CapabilityStatement/CapabilityStatement-genomics-requirements.json

```diff
@@ -129,7 +129,7 @@
                 },
                 {
                     "type": "RelatedPerson",
-                    "profile": "https://fhir.hl7.org.uk/StructureDefinition/UKCore-RelatedPerson"
+                    "profile": "https://fhir.nhs.uk/StructureDefinition/NHSDigital-RelatedPerson-Genomics"
```

Comment:
Added new profile for RelatedPerson to support extensions, covering requirements from Genomics